Frontend/tailwind.config.js

```diff
@@ -2,7 +2,22 @@
 export default {
   content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
   theme: {
-    extend: {},
+    extend: {
+      fontFamily: {
+        Poppins: ['Poppins', 'sans-serif'],
```

Comment:
lets use small letters all through for font properties (poppins,lato) and camelcase for the AtypeFont. Should be aTypeFont.